_includes/_section-projects.html

```diff
@@ -19,6 +19,13 @@ <h3>{{ project.title }}</h3>
           <p>{{ project.category }}</p>
         </div>
         {% endif %}
+        <div class="project-membership">
+          {% if project.affiliate %}
+          <img src="https://img.shields.io/badge/typelevel-affiliate-FFB4B5" alt="Typelevel affiliate" />
+          {% else %}
+          <img src="https://img.shields.io/badge/typelevel-member-FF6169" alt="Typelevel member" />
+          {% endif %}
+        </div>
```

Comment:
This should be DRYed up with the other.  This whole section could, perhaps.  I'm running out of jekyll fucks.

Reply:
Agreed, but refactoring can come at a later date.